shared/video-sdk/reference/_pricing.mdx

```diff
@@ -148,6 +148,30 @@ Agora offers a 10% overage allowance for all tiers of PCU at no additional charg
 
 Billing occurs at the end of each calendar month. For other tiers, contact [Agora Customer Support](mailto:support@agora.io).
 
+### AI Noise Suppression pricing
+
+[AI Noise Suppression](../enable-features/ai-noise-suppression) helps reduce noise and voice distortion in virtual communication.
+Agora charges for voice sent to the channel by the host while noise suppression is activated.
+
+AI Noise Suppression is free of charge until March 31, 2023. Starting from April 1, 2023, the prices are as follows:
+
+|Usage, minutes per month |Pricing, US$/1,000 minutes|
+|------------------|--------------------------|
+|0 to 10,000       |Free		              |
+|Above 10,000      |0.59                      |
+
+### 3D Spatial Audio pricing
+
+[3D Spatial Audio](../enable-features/spatial-audio) brings theater-like effects to virtual communication, making it seem as if the sound originates
+from all around the user. Agora charges for the time users spend in a channel with spatial audio activated.
```

Comment:
Do you mean each user? That would make a difference to the price. 

while 3D Spatial Audio is activated.

Reply:
Yeah, I assume it's each user, fixing